lib/ts/recipe/session/with-jwt/recipeImplementation.ts

```diff
@@ -158,8 +174,16 @@ export default function (
 
             let existingJWTValidity = decodedPayload.exp - currentTimeInSeconds;
 
+            let defaultClaimsToAdd = {
+                sub: sessionInformation.userId,
+                iss: appInfo.apiDomain.getAsStringDangerous(),
+            };
```

Comment:
Why do you need to add these here too? Shouldn't this already be in the access token payload cause of the create new session function?

Reply:
This is because updateAccessTokenPayload should assume that the old payload is no longer relevant and treat the new payload as it would in create session. (To prevent older sub and iss values from being persisted)